region/tourian/main/Seaweed Room.json

```diff
@@ -680,18 +772,27 @@
       "flashSuitChecked": true
     },
     {
-      "id": 34,
       "link": [3, 1],
-      "name": "Come In Shinecharged, Leave With Spark (HiJump Wall Jump)",
+      "name": "Come In Shinecharged, Leave With Spark",
       "entranceCondition": {
         "comeInShinecharged": {}
       },
       "requires": [
-        {"shineChargeFrames": 100},
-        "HiJump",
-        "canWalljump",
-        "canShinechargeMovementComplex",
-        {"shinespark": {"frames": 5}}
+        "Wave",
+        {"or": [
+          {"and": [
+            "HiJump",
+            "canWalljump",
+            {"shineChargeFrames": 100},
```

Comment:
I can't get 100 frames with hijump+walljump.  And my best Hijump only is 1 frame slower than my best hi+wall, but hijump only is a good deal harder to do fast.

Reply:
Here's a video showing getting 93 frames with HiJump+walljump: https://videos.maprando.com/video/6938. My best is 91.

Reply:
That may be the problem.  I was doing "Carry Shinecharge", above this.